eng/.docsettings.yml

```diff
@@ -115,6 +115,7 @@ known_content_issues:
   - ['sdk/authorization/microsoft-azure-authentication-msi-token-provider/README.md', '#3113']
   - ['sdk/batch/microsoft-azure-batch/README.md', '#3113']
   - ['sdk/boms/azure-sdk-bom/README.md', '#3113']
+  - ['sdk/boms/azure-external-dependencies-bom/README.md', '#3113']
```

Comment:
Now that we have combined Azure client libraries and external dependencies into a single bom, this is no longer required.

Reply:
Removed.